File: embedded/r/fsda_embedded.R
library(reticulate)

use_python("./venv/Scripts/python.exe", required = TRUE)
source_python("./src/gateways/fsda_gateway.py")

fsda <- FSDA()

data <- matrix(c(1,2,3,4,5,6,7,8,9), nrow=3, byrow=TRUE)
result <- fsda$run("zscoreFS", data)
print(result)

File: embedded/julia/fsda_embedded.jl
using PyCall

gateway_path = joinpath(pwd(), "src", "gateways")
pushfirst!(PyVector(pyimport("sys")."path"), gateway_path)

fsda_module = pyimport("fsda_gateway")
fsda = fsda_module.FSDA()

result = fsda.run("zscoreFS", [1.0 2.0 3.0; 4.0 5.0 6.0; 7.0 8.0 9.0])
println(result)